### PR TITLE
update two existing internet-health pages per PR 4695

### PR DESCRIPTION
**Pull request by peiying2**
Updated files:
- internet-health.lang
- internet-health/privacy-security.lang
- mozorg/newsletters.lang

Add new file: digital-inclusion.lang

### CODE REVIEW

File: en-US/mozorg/internet-health/privacy-security.lang [hunk 1]
```diff
@@ -57,14 +67,33 @@ We should all be able to choose – with clarity and confidence – what informa
...
 ;Make sure your mobile apps access only the info they need. Control your privacy and location settings on <a href="%(ios)s" rel="external">iOS</a> and <a href="%(android)s" rel="external">Android</a>.
 Make sure your mobile apps access only the info they need. Control your privacy and location settings on <a href="%(ios)s" rel="external">iOS</a> and <a href="%(android)s" rel="external">Android</a>.
 
 
+##TAG: internet_health_subpage_rebrand
+;Manage your preferences
+Manage your preferences
+
+
+;Know your settings. You can manage your profile and preferences for <a href="{{ google }}">Google</a>, <a href="{{ yahoo }}">Yahoo!</a> and <a href="{{ facebook }}">Facebook</a> ads.
```
Comment (TheoChevalier) [inline en-US/mozorg/internet-health/privacy-security.lang]: You’ll need to replace variables with `%(google)s`, `%(privacy)s` and `%(facebook)s`, that’s what bedrock will expect

File: en-US/mozorg/internet-health/privacy-security.lang [hunk 2]
```diff
@@ -182,10 +285,15 @@ Cohosting <a href="%(standford)s" rel="external">talks with Stanford’s Center
 Hungry for more?
 
 
+# This string is obsolete. Don't remove.
 ;Keep reading about Privacy and Security in our <a href="%(healthreport)s">Internet Health Report</a>.
 Keep reading about Privacy and Security in our <a href="%(healthreport)s">Internet Health Report</a>.
 
 
+;Read on about Digital Inclusion in our <a href="{{ healthreport }}">Internet Health Report</a>.
```
Comment (TheoChevalier) [inline en-US/mozorg/internet-health/privacy-security.lang]: Same here `%(healthreport)s`

File: en-US/mozorg/internet-health/privacy-security.lang (hunk 1 above)
Comment (peiying2) [inline en-US/mozorg/internet-health/privacy-security.lang]: Thanks for noticing, fixed them in my next push

**Comment by peiying2** [comment]
@flodolo there are a few of the same strings shared in both privacy-security and digital-inclusion files.  I am not sure if we should move them to main.lang.  There are a few more pages like these in development. They may have the same verbiage.

File: en-US/mozorg/newsletters.lang [hunk 3]
```diff
@@ -19,6 +20,15 @@ Select country
 Country:
 
 
+## TAG: mozilla_newsletter_2017
+;Subscribe to updates and keep current with Mozilla news. It’s the perfect way for us to keep in touch!
+Subscribe to updates and keep current with Mozilla news. It’s the perfect way for us to keep in touch!
+
+
+;Subscribe to monthly updates and keep current with Mozilla news, including the latest tips and tricks for getting the most out of your Firefox browser. It’s the perfect way for us to keep in touch!
```
Comment (flodolo) [inline en-US/mozorg/newsletters.lang]: Interesting, this string is in /newsletter.lang

Let's not add the old string here, only the new one.

File: en-US/mozorg/internet-health/privacy-security.lang [hunk 4]
```diff
@@ -57,14 +67,33 @@ We should all be able to choose – with clarity and confidence – what informa
...
 ;Make sure your mobile apps access only the info they need. Control your privacy and location settings on <a href="%(ios)s" rel="external">iOS</a> and <a href="%(android)s" rel="external">Android</a>.
 Make sure your mobile apps access only the info they need. Control your privacy and location settings on <a href="%(ios)s" rel="external">iOS</a> and <a href="%(android)s" rel="external">Android</a>.
 
 
+## TAG: internet_health_subpage_rebrand
+;Manage your preferences
+Manage your preferences
+
+
+;Know your settings. You can manage your profile and preferences for <a href="%(google)s">Google</a>, <a href="%(yahoo)s">Yahoo!</a> and <a href="%(facebook)s">Facebook</a> ads.
```
Comment (flodolo) [inline en-US/mozorg/internet-health/privacy-security.lang]: Missing tag here

File: en-US/mozorg/internet-health/privacy-security.lang [hunk 5]
```diff
@@ -57,14 +67,33 @@ We should all be able to choose – with clarity and confidence – what informa
...
+Know your settings. You can manage your profile and preferences for <a href="%(google)s">Google</a>, <a href="%(yahoo)s">Yahoo!</a> and <a href="%(facebook)s">Facebook</a> ads.
+
+
 ;Know your settings. You can manage your profile and preferences for <a href="%(google)s">Google</a>, <a href="%(yahoo)s">Yahoo!</a> and <a href="%(facebook)s">Facebook</a> ads, and even <a href="%(acxiom)s">edit data that’s been collected about you by Acxiom</a>, one of the world’s largest marketing data brokers.
```
Comment (flodolo) [inline en-US/mozorg/internet-health/privacy-security.lang]: Add usual note (obsolete, do not remove)

File: en-US/mozorg/internet-health/privacy-security.lang [hunk 6]
```diff
@@ -133,6 +201,10 @@ Educating the industry about <a href="%(leandata)s">lean data practices</a>.
 Government Surveillance: Keeping prying eyes and ears out of your business
 
 
+;We should all have the freedom to be ourselves — online and off – without surveillance, judgment and imposed societal bias.
```
Comment (flodolo) [inline en-US/mozorg/internet-health/privacy-security.lang]: The same string is already available 4 lines later.

File: en-US/mozorg/internet-health/privacy-security.lang [hunk 7]
```diff
@@ -182,10 +285,15 @@ Cohosting <a href="%(standford)s" rel="external">talks with Stanford’s Center
 Hungry for more?
 
 
+# This string is obsolete. Don't remove.
 ;Keep reading about Privacy and Security in our <a href="%(healthreport)s">Internet Health Report</a>.
 Keep reading about Privacy and Security in our <a href="%(healthreport)s">Internet Health Report</a>.
 
 
+;Read on about Digital Inclusion in our <a href="%(healthreport)s">Internet Health Report</a>.
```
Comment (flodolo) [inline en-US/mozorg/internet-health/privacy-security.lang]: This string should have been behind a tag. Add the tag `internet_health_subpage_rebrand`, I'll fix the code.

File: en-US/mozorg/internet-health/privacy-security.lang [hunk 8]
```diff
@@ -57,14 +67,33 @@ We should all be able to choose – with clarity and confidence – what informa
...
+# This is used as a decorative element in a callout box.
+;91%
+91%
+
+
 ;What you can do about user control
 What you can do about user control
 
 
+## TAG: internet_health_subpage_rebrand
```
Comment (flodolo) [inline en-US/mozorg/internet-health/privacy-security.lang]: This tag is not available at the beginning of the file, you need to add it there two.

**Comment by flodolo** [comment]
Side note: you want to import strings from internet-health/privacy-security.lang for the new template, since there are strings in common.

**Comment by flodolo** [comment]
https://github.com/mozilla/bedrock/pull/4702/files

There's another string for the menu ("Open Innovation"), and a third page. Not sure if it's worth waiting for that to be done?

Side question, given the error with placeholders: did you extract the strings or copied them from the pull request?

File: en-US/mozorg/internet-health/privacy-security.lang [hunk 9]
```diff
@@ -182,10 +285,15 @@ Cohosting <a href="%(standford)s" rel="external">talks with Stanford’s Center
 Hungry for more?
 
 
+# This string is obsolete. Don't remove.
```
Comment (flodolo) [inline en-US/mozorg/internet-health/privacy-security.lang]: Turns out this string is not obsolete

File: en-US/mozorg/internet-health/privacy-security.lang (hunk 7 above)
Comment (flodolo) [inline en-US/mozorg/internet-health/privacy-security.lang]: You can remove this string, it doesn't belong to this file

File: en-US/mozorg/internet-health.lang [hunk 10]
```diff
@@ -52,6 +52,7 @@ Learn more
 Discover open innovation
 
 
+# This string is obsolete.  Do not remove.
```
Comment (flodolo) [inline en-US/mozorg/internet-health.lang]: Need to remove this comment, the string is used and not obsolete (see https://github.com/mozilla/bedrock/pull/4699)

File: en-US/mozorg/internet-health/privacy-security.lang [hunk 11]
```diff
@@ -1,3 +1,6 @@
+## internet_health_open_innovation ##
```
Comment (flodolo) [inline en-US/mozorg/internet-health/privacy-security.lang]: Remove this tag, not sure where you took it from

File: en-US/mozorg/internet-health/privacy-security.lang [hunk 12]
```diff
@@ -12,6 +15,20 @@ A Healthy Internet is Secure and Private
...
+;Privacy &amp; Security
+Privacy &amp; Security
+
+
+## TAG: internet_health_digital_inclusion
+;Digital Inclusion
+Digital Inclusion
+
+
+## TAG: internet_health_open_innovation
```
Comment (flodolo) [inline en-US/mozorg/internet-health/privacy-security.lang]: This string is behind the same tag as the rest of the menu (internet_health_digital_inclusion)
https://github.com/alexgibson/bedrock/blob/06448a41063b2d870df9f018324b8db2ccd3326a/bedrock/mozorg/templates/mozorg/internet-health/privacy-security.html#L29-L37

File: en-US/mozorg/internet-health/digital-inclusion.lang [hunk 13]
```diff
@@ -0,0 +1,265 @@
+## internet_health_open_innovation ##
```
Comment (flodolo) [inline en-US/mozorg/internet-health/digital-inclusion.lang]: Remove this tag

File: en-US/mozorg/internet-health/digital-inclusion.lang [hunk 14]
```diff
@@ -0,0 +1,265 @@
...
+
+;Privacy &amp; Security
+Privacy &amp; Security
+
+
+;Digital Inclusion
+Digital Inclusion
+
+
+## TAG: internet_health_open_innovation
```
Comment (flodolo) [inline en-US/mozorg/internet-health/digital-inclusion.lang]: Remove this tag, there are no tags for the menu in this template
https://github.com/alexgibson/bedrock/blob/06448a41063b2d870df9f018324b8db2ccd3326a/bedrock/mozorg/templates/mozorg/internet-health/digital-inclusion.html#L24-L31

**Pull request merged**